packages/core/src/2d/text/TextStyle.ts

```diff
@@ -0,0 +1,63 @@
+import { assignmentClone } from "../../clone/CloneManager";
+import { UpdateFlag } from "../../UpdateFlag";
+import { UpdateFlagManager } from "../../UpdateFlagManager";
+
+export class TextStyle {
```

Comment:
Maybe a litte heavy, If TextStyle is a Enum. We can use:
```ts
 textRenderer.textStyle= TextStyle.italic | TextStyle.bold;
```

Reply:
done